File: Java/test/commando/SumOfSquaresCommandTest.java
package commando;

import org.junit.Test;

import static org.hamcrest.MatcherAssert.assertThat;
import static org.hamcrest.Matchers.equalTo;

public class SumOfSquaresCommandTest {

    private final SumOfSquaresCommand command = new SumOfSquaresCommand();

    @Test
    public void willCalculateSumOfSquares() throws Exception {
        assertThat(command.execute("3,4"), equalTo("25"));
    }

    @Test (expected = InvalidCommandException.class)
    public void willBlowUpIfIllegalFormat() throws Exception {
        command.execute("a,4");
    }
}

File: PHP/test/Commando/SumOfSquaresCommandTest.php
<?php
namespace Tests;

use PHPUnit\Framework\TestCase;
use Commando\SumOfSquaresCommand;
use Commando\InvalidCommandException;

class SumOfSquaresCommandTest extends TestCase {

    private $command;

    protected function setUp(): void {
        $this->command = new SumOfSquaresCommand();
    }

    /** @test */
    public function willCalculateSumOfSquares() {
        $this->assertEquals("25", $this->command->execute("3,4"));
    }

    /** @test */
    public function willBlowUpIfIllegalFormat() {
        $this->expectException(InvalidCommandException::class);
        $this->command->execute("a,4");
    }
}
